test/torch_test_meta.py

```diff
@@ -155,6 +155,7 @@
     'test_triangular_solve',  # (TPU) precision (1e-7)
     'test_scalar_check',  # runtime error
     'test_argminmax_large_axis',  # OOM, and the test is grepping "memory" in the exception message
+    'test_trapz' # precision (1e-5)
```

Comment:
I tested this right now, and the test is green.
Seem strange as the default precision is:
```Python
DEFAULT_FLOATING_PRECISION = 1e-3
```
In `torch_test_meta.py`.
Maybe `test_trapz` uses `normal*()` and this lead to out-of-precision errors.
Can you try to fiddle with `torch_test_precisions` in `torch_test_meta.py`?
Even though the 1e-3 default should do it ... assuming it actually works.

Reply:
I was wrong about the 1e-5 part, it should be 1e-3.

```
(Pdb) p expected
array([-7.32792318e-03,  1.01249252e+01])
(Pdb) p actual
tensor([-7.3280e-03,  1.0125e+01], device='xla:0')
(Pdb) self.precision
0.001
(Pdb) self.assertEqual(expected, actual.)
(Pdb) self.assertTrue(np.allclose(expected, actual.cpu().numpy()))
AssertionError: False is not true
```

trapz does not use normal, however the test use np.allClose which has the defualt rtol 1e-5. I will update the comment.

Reply:
Will fix in next pr